src/main/java/email_request/MatcherUITemplate.java

```diff
@@ -16,20 +12,24 @@ public class MatcherUITemplate extends JPanel implements ActionListener {
     private String targetUserEmail;
 
     public MatcherUITemplate(MatcherUIController matcherUIController,
-                             String requesterEmail, String targetUserEmail,
-                             MatchResponseModel matchResponseModel){
+                             String requesterEmail, String targetUserEmail){
 
         this.matcherUIController = matcherUIController;
         this.requesterEmail = requesterEmail;
         this.targetUserEmail = targetUserEmail;
 
-        for(UserData matchResults: matchResponseModel.getMatchedData()){
-            MatchUserPanel matchUserPanel = new MatchUserPanel(matchResults);
-            this.add(matchUserPanel);
-        }
+        JLabel title = new JLabel("Profile");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JButton requestEmail = new JButton("Request Email");
+
+        JPanel buttons = new JPanel();
+        buttons.add(requestEmail);
+
+        requestEmail.addActionListener(this);
```

Comment:
The previous code will have border line between each small panel as it combined multiple small panel with boarder into a bigger panel. In the new version, the whole panel will only have one boarder, which means that the division line between each user information will disappear. To restore the division line, changes need to be made in MatcherUITemplateJames.

Reply:
I changed code to add division lines between each profile, and added two more buttons for "report" and "block" a user profile.